test/haz3ltest.re

```diff
@@ -3,7 +3,10 @@ open Junit_alcotest;
 let (suite, _) =
   run_and_report(
     ~and_exit=false,
-    "Dynamics",
-    [("Elaboration", Test_Elaboration.elaboration_tests)],
+    "HazelTests",
```

Comment:
It's still very unclear to me at what level this report is supposed to be built at. A lot of sample repos online just have the full suite or tests. The alcotest documentation lists a module but no clear way to have a bunch of modules in a report.